packages/ui/src/components/avatar/brc20-avatar-icon.web.tsx

```diff
@@ -1,51 +1,57 @@
+import { styled } from 'leather-styles/jsx';
```

Comment:
Maybe we can put these into a folder like `lib` / `instances` or something? Then we have a clean interface to the base component - `avatar/avatar` , `avatar/lib/brc20...` 

I'm not sure on the naming but it has historically been an avatar dumping ground in here. 

Maybe the issue is that they need to be refactored and have the `icon` passed in the same as the other avatars?

Reply:
I'd frankly even take this further and move those into their respective apps altogether, I don't think this is the right place for these domain-specific icons to live. These also are sort of special cases, we're abusing the `icon` prop to display what in reality is an image that should be passed into Avatar at the usage point.
Had plans to do this initially, but with all the extension changes the PR was snowballing already.
Can we make an issue on improved handling token images? Might be worth including the fallback/image alt improvements too.

Reply:
Sounds great 👍 a future improvement issue is fine

Reply:
Done: [LEA-2122](https://linear.app/leather-io/issue/LEA-2122/issues-with-token-avatar-usage)